docs/docs_style_guide.md

```diff
@@ -0,0 +1,479 @@
+
+### Ampersand
+
+Use 'and' rather than an '&', unless it's part of syntax.
+
+## B
+
+### Brackets
+
+Use (round brackets), not [square brackets] in documentation text.
+
+Don't use round brackets to refer to something that could either be
+singular or plural, eg 'Copy the object(s) to a new stack.'
+
+Always use the plural instead as this will cover each possibility, eg
+'Copy the objects to a new stack.'
+
+### Bullets and steps
+
+#### Bullet points
+
+You can use bullet points to make text easier to read. Make sure that:
+
+* you always use a lead-in line
+* the bullets make sense running on from the lead-in line
+* you use lower case at the start of the bullet
+* you don't use more than one sentence per bullet point - use commas,
+  dashes or semicolons to expand on an item
+* you don't put 'or', 'and' after the bullets
+* if you add links they appear within the text and not as the whole
+  bullet
+* there is no full stop after the last bullet point
+
+#### Steps
+
+Use numbered steps instead of bullet points to guide a user through a
+process. You don't need a lead-in line and you can use links and
+downloads (with appropriate Markdown) in steps. Each step ends in a
+full stop because each step should be a complete sentence.
+
+## C
+
+### Capitalisation
+
+DON'T USE BLOCK CAPITALS FOR LARGE AMOUNTS OF TEXT AS IT'S QUITE HARD
+TO READ.
+
+### Contractions
+
+Use contractions, eg 'they've', 'we'll'. Avoid using 'should've',
+'could've', 'would've' etc - these are hard to read.
+
+### Dates
+
+Use upper case for months, eg 'January', 'February'.
+
+Don't use a comma between the month and year, eg '14 June 2012'
+
+When space is an issue, eg tables, titles etc, you can use truncated
+months, eg 'Jan, Feb, Mar, Aug, Sept, Oct, Nov, Dec'.
+
+Use 'to' in date ranges - not hyphens, en rules or em dashes. For
+example:
+
+* 'Monday to Friday, 9am to 5pm' (put different days on a new line,
+  don't separate with a comma etc)
+
+* 10 November to 21 December
+
+Don't use 'quarter' for dates; use the months, for example: 'Jan to
+Mar 2013'.
+
+## E
+
+### eg, etc and ie
+
+Don't use full stops after or between these notatinos.
+
+You can use the long form ('for example' instead of 'eg',
+'specifically' instead of 'ie' etc) if you like. Some people are not
+familiar with abbreviations such as eg, so consider your audience
+before abbreviating.
+
+### email
+
+One word.
+
+### Email addresses
+
+Write email addresses in full, in lower case and as active links.
+Don't include any other wards as part of the link. Don't label email
+addresses ie don't write 'Email: name@example.org'.
+
+### etc
+
+See 'eg, etc and ie'.
+
+## F
+
+### FAQs (frequently asked questions)
+
+Don't use FAQs in LiveCode documentation. If you write content by
+starting with user needs, you won't need to use FAQs.
+
+## H
+
+### homepage
+
+Lower case.
+
+### Hyphenation
+
+Hyphenate:
+
+* 're-' words starting with 'e', eg 're-evaluate'
+* co-ordinate
```

Comment:
Maybe it's a UK thing, but co-ordinate and co-operate just seem weird to me. Particularly co-operate makes me think of one of those lumberjack saws that needs two people.